mondey_backend/src/mondey_backend/models/questions.py

```diff
@@ -44,6 +45,7 @@ class QuestionPublic(SQLModel):
     additional_option: str = ""
     required: bool = False
     name: str = ""
+    visibility: bool = False
```

Comment:
since the backend only returns `QuestionPublic` objects if they are visible, I don't think we need to include this here - visibility is not something the frontend should care about, it just displays whatever the backend gives it
```suggestion
```